sys-libs/readline/files/readline-8.1-rlfe-c99.patch

```diff
@@ -0,0 +1,291 @@
+From d09284d53d1a4d1d58207b1240986329285d01ca Mon Sep 17 00:00:00 2001
```

Comment:
Have you sent this to Chet (ideally on bug-readline)? If so, include a link. If not, please do ;)

Reply:
OK, just send an email reporting it to him then.